### ISSUE TITLE
Prerequisites for thumbnail generation

### ISSUE THREAD

**Issue by joshcarst** [meganz/MEGAcmd#742]
Hi,

I'm running Ubuntu 20.04. I'm trying to get video thumbnail generation to work. I first installed MEGAcmd via the .deb package on MEGA's website without installing ffmpeg and videoinfo prior which didn't generate thumbnails, so I removed my installation with apt-get remove and tried building it from git source this time after installing those dependencies. My session and login info seemed to be retained from my first installation with the new build and thumbnails are still not generating, and using the mega-thumbnail command to assign a locally-generated image as a thumbnail just hangs indefinitely. Is there some extra uninstallation step I'm missing to make sure I build MEGAcmd with ffmpeg/mediainfo correctly? Running mega-thumbnail doesn't generate any logs even with the -v flag, unsure how to generate logs elsewhere.

Thanks!

**Comment by polmr** [comment]
Hi @joshcarst , there is a known issue with some video files in Ubuntu 20.04 packages. However, building with system's ffmpeg should broaden the scope of supported formats :thinking: ...
```
ldd `which mega-cmd-server` | grep av
```
should list ffmpeg libraries in case it was built linking dynamically with them.

```
mega-thumbnail -s /path/in/mega.video /path/to/local/thumbnail.png
```
should not halt in any case.
This:
https://github.com/meganz/MEGAcmd/blob/master/contrib/docs/DEBUG.md#linux
indicates how to obtain logs that may give us a hint on that's going on.

Btw, notice that in order to generate a thumbnail, MEGAcmd will need the file to be uploaded anew. Meaning that if there is a remote copy of the file, the thumbnail generation won't be triggered. 
some:
```
touch /path/to/local/vid.eo
``` 
would prevent a remote match and force that.

**Comment by joshcarst** [comment]
So it turns out my issue had the same fix as #252 where I just had to run `sudo ldconfig`. My mega-cmd-server was previously missing a lot of library links, one of which was libmega.so as well as all the ffmpeg links. Thumbnails are generating now. Thanks for the nudge!
